### DISCUSSION TITLE
Task 1: Metrics evaluation

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#87] [General]
Which is the most important metric from the following? 

Binary DICE WT, Binary DICE ET, Binary DICE TC, Hausdorff95 WT, Hausdorff95 ET, Hausdorff95 TC

For the submission (from what I understand), All metrics + communication cost is considered separately and Teams get 7 different scores? (Is there any other metric?) Team with most high scores is considered higher on the ranking?

**Comment by sarthakpati** [comment]
There is no "most important" metric. All of them will be considered for the final ranking.
